test/DynamicEater.test.ts

```
@@ -98,10 +98,28 @@ describe("Dynamic Merge / Eater Contract", function () {
-		it("Should add user to white list", async () => {
+		it.only("Should be able to merge when merkle root is bytes 0", async () => {
 			const { merge, otherAccount } = await loadFixture(deployFixture);
 
-			await merge.addWhiteList(otherAccount.address, ethers.parseUnits("1", 9));
+			await merge.deposit(ethers.parseEther("10000"));
+
+			const amount = ethers.parseUnits("100", decimals);
+			await expect(merge.connect(otherAccount).merge(amount)).to.emit(merge, "Merged");
+		});
+
+		it.only("Should be able to merge with proof", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)